src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -8,7 +8,7 @@ import { addComments } from '../../services/comments';
 
 type Props = {
   selectedPost: Post;
-  setComments: (value: Comment[] | { (prev: Comment[]): Comment[] }) => void,
+  setComments: (value: Comment[] | ((prev: Comment[]) => Comment[])) => void;
```

Comment:
read provided link once more [here](https://stackoverflow.com/a/65824149) the correct answer

Reply:
like that?
`setComments: React.Dispatch<React.SetStateAction<Comment[]>>;`

Reply:
> like that? `setComments: React.Dispatch<React.SetStateAction<Comment[]>>;`
 yes, it is more correct type. use in such cases in next tasks